src/components/WalletModal/WalletView/index.tsx

```diff
@@ -73,9 +81,16 @@ export default function WalletView({
 
     if (isLoading) {
       return (
-        <Box display="flex" marginTop="20px" alignItems="center">
-          <Loader size="24px" />
-          <Text color="text1">{t('walletModal.initializing')}</Text>
+        <Box display="flex" marginTop="20px" alignItems="center" flexDirection="column" style={{ gap: 20 }}>
+          {qrCodeUri ? (
+            <QRCodeBox>
+              <QRCode value={qrCodeUri} />
+            </QRCodeBox>
+          ) : null}
+          <Box display="flex" alignItems="center">
+            <Loader size="24px" />
+            <Text color="text1">{t('walletModal.initializing')}</Text>
+          </Box>
```

Comment:
1. Ok I will fix this issue, about initializing it's purposeful because is showing to user is waiting to connect to wallet
2. I will fix
3. it's not a bug it's showing that you are connected with metamask in the app, if the user wants to change the network he must change in the wallet or in the networking selection, the wallet modal will show which wallet you are connected

https://github.com/pangolindex/components/assets/37405304/32819ca6-f823-435f-b00c-6a6794b10485